packages/invoices/src/InboundShipment/DetailView/modals/InboundLineEdit/InboundLineEdit.tsx

```diff
@@ -14,12 +14,12 @@ import {
   DialogButton,
   useDialog,
   generateUUID,
-  Item,
```

Comment:
Inbounds in general: Was using the `Item` type for the edit modal - I just changed that to be `ItemRowFragment`